service/conf/application.conf

```diff
@@ -480,6 +485,11 @@ thisActorSystem {
             nr-of-instances = 5
             dispatcher = group-dispatcher
           }
+        /CacheActor{
+          router = smallest-mailbox-pool
+          nr-of-instances = 5
```

Comment:
Group-dispatcher actor instances need some higher numbers.
read - 15
cache - 15

create - 10
search - 10

Reply:
done and added dispatcher in each Actor